monitoring_prometheus/models/ir_http.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+from odoo.http import request
+from prometheus_client import Summary, Counter
+
+
+REQUEST_TIME = Summary(
+    "request_latency_sec", "Request response time in sec", ["query_type"]
+)
+LONGPOLLING_COUNT = Counter("longpolling", "Longpolling request count")
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = "ir.http"
+
+    @classmethod
+    def _dispatch(cls):
+        path_info = request.httprequest.environ.get("PATH_INFO")
+
+        if path_info.startswith("/longpolling/"):
+            LONGPOLLING_COUNT.inc()
+            return super()._dispatch()
```

Comment:
Compatible Python 2.7?
```suggestion
            return super(IrHttp, cls)._dispatch()
```

And also the 2 other occurrences below